### DISCUSSION TITLE
Welcome to Sike for Java Discussions!

### DISCUSSION THREAD

**Discussion by petrdvorak** [wultra/sike-java#50] [General]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:

- Ask questions you’re wondering about.
- Share ideas.
- Engage with other community members.

**Comment by bjanuario** [comment]
How can we test an attempt failure to hack the message ?

**Reply by petrdvorak** [reply]
Hello @bjanuario, you can download the sources, build them, and test anything you would like to test. Unfortunately, we are not able to provide any assistance with such activities.
